pkg/controller/network/nsnetworkpolicy/nsnetworkpolicy_controller.go

```diff
@@ -301,7 +303,7 @@ func (c *NSNetworkPolicyController) generateNodeRule() (netv1.NetworkPolicyIngre
 	return rule, nil
 }
 
-func generateNSNP(workspace string, namespace string, matchWorkspace bool) *netv1.NetworkPolicy {
+func (c *NSNetworkPolicyController) generateNSNP(workspace string, namespace string, matchWorkspace bool) *netv1.NetworkPolicy {
```

Comment:
Please add some unit test for generateNSNP

Reply:
This function has already been unit tested.

Reply:
OK, I see